storage_scrubber/src/lib.rs

```diff
@@ -242,24 +242,39 @@ impl ConsoleConfig {
     }
 }
 
-pub fn init_logging(file_name: &str) -> WorkerGuard {
-    let (file_writer, guard) =
-        tracing_appender::non_blocking(tracing_appender::rolling::never("./logs/", file_name));
-
-    let file_logs = fmt::Layer::new()
-        .with_target(false)
-        .with_ansi(false)
-        .with_writer(file_writer);
-    let stderr_logs = fmt::Layer::new()
-        .with_target(false)
-        .with_writer(std::io::stderr);
-    tracing_subscriber::registry()
-        .with(EnvFilter::try_from_default_env().unwrap_or_else(|_| EnvFilter::new("info")))
-        .with(file_logs)
-        .with(stderr_logs)
-        .init();
-
-    guard
+pub fn init_logging(file_name: &str) -> Option<WorkerGuard> {
+    let disable_file_logging = match std::env::var("PAGESERVER_DISABLE_FILE_LOGGING") {
+        Ok(s) => s == "1" || s.to_lowercase() == "true",
+        Err(_) => false,
+    };
+
+    if disable_file_logging {
+        let stderr_logs = fmt::Layer::new()
```

Comment:
this variable could be moved outside of the `if`, it would save on duplication.

Reply:
i'll fix that once i get a docker image for the current patch :)